dev-libs/hyperscan/hyperscan-5.4.0.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
+
+inherit cmake flag-o-matic python-any-r1
+
+DESCRIPTION="High-performance regular expression matching library"
+SRC_URI="https://github.com/intel/hyperscan/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+HOMEPAGE="https://www.hyperscan.io/ https://github.com/intel/hyperscan"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cpu_flags_x86_avx2 +cpu_flags_x86_ssse3 static-libs"
+
+RDEPEND="dev-libs/boost"
```

Comment:
:= for anything we link against which exposes a subslot, but especially Boost

Reply:
Ok, thanks.